custom/ewsghana/api.py

```diff
@@ -332,11 +335,25 @@ def _set_up_supply_point(self, location, ews_location):
             location.name = supply_point.name
             location.site_code = supply_point.code
             location.location_type = supply_point.type
-            self._create_supply_point_from_location(supply_point, location)
+            try:
+                sql_location = SQLLocation.objects.get(domain=self.domain, site_code=supply_point.code)
```

Comment:
how do you know that you are clearing the code from a district? is it just based on knowing the order that these are called?